README.md

```diff
@@ -37,3 +37,32 @@ allow re-use when changing the UI framework.
 You can then run the tests using
 
     npm test
+
+# Configurations to run the application
+
+The application relies on cloud communication and updates from actual robots to
+work correctly.
+
+1. First of all you need an AWS account, and you need to create a user to get an
+   AWS Access Key ID, AWS Secret Access Key and region.
+
+2. Then you need to clone and deploy the backend,
+   https://github.com/NordicPlayground/robot-war-cloud
+
+3. Then you can go to the Settings page on this application, and enter your AWS
+   Access Key ID and AWS Secret Access Key.
+
+4. To display the robots on the app, you need actual robots communicating with
+   the cloud. To get around this, you can simulate robots by following the next
+   steps
+
+#### To be able to run the application without robots:
```

Comment:
As a suggestion, I think this title could be `Run application with simulate robots` instead.